src/components/calculator.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Calculator extends React.Component {
+  render() {
+    return (
+      <main>
```

Comment:
- [ ] Great job working on the Calculator so far but I think it would be great if you make your component as functional based component instead of class based component as required in the requirements 💯🔥🤩🙏🏻

![image](https://user-images.githubusercontent.com/58553711/229970835-d5b7b901-5dfd-400d-9859-61b66186d192.png)

Reply:
Thank you! I actually had it that way first but compiling it gave so many errors. I found a fix though hehe. Thanks again!